docs/tutorial.rst

```diff
@@ -1,4 +1,620 @@
+   }
+
+   div.timezone {
+     width: 45%;
+   }
+
+   p.info {
+     font-size: 2em;
+     line-height: 2;
+   }
+
+   time {
+     color: #ff0000;
+   }
+
+
+The changes to the application code will be quite small.
+First, we define the file system path to the folder
+that contains the static assets:
+
+.. code-block:: python
+
+   CUR_PATH = os.path.dirname(os.path.abspath(__file__))
+   STATIC_PATH = os.path.join(CUR_PATH, "static")
+
+
+And then we add a route by creating
+a :class:`StaticApplication <clastic.static.StaticApplication>`
+with the static file system path we have defined,
+and we set it as the endpoint that will handle the requests
+to any application path under ``/static``:
+
+.. code-block:: python
+   :emphasize-lines: 4, 8
+
+   # from clastic.static import StaticApplication
+
+   def create_app():
+       static_app = StaticApplication(STATIC_PATH)
+       routes = [
+           ("/", home, "home.html"),
+           ("/show", show_time, "show_time.html"),
+           ("/static", static_app),
+       ]
+       render_factory = AshesRenderFactory(CUR_PATH)
+       return Application(routes, render_factory=render_factory)
+
+
+Don't forget to add the stylesheet link to the templates:
+
+.. code-block:: html
+
+   <head>
+     <meta charset="utf-8">
+     <title>Time zone convertor</title>
+     <link rel="stylesheet" href="/static/custom.css">
+   </head>
+
+
+Working with JSON
+-----------------
+
+In the last part of the tutorial,
+we're going to display the converted time
+in the same page as the form instead of moving to a second page.
+In order to achieve this,
+we're going to use JavaScript to update the page
+with data sent to and received from the application using JSON.
+
+First, we're going to modify our ``show_time()`` endpoint function.
+Note that the only difference is
+that the submitted data are loaded from ``request.data``
+instead of being accessed through ``request.values``,
+and that there is no change in the returned value:
+
+.. code-block:: python
+   :emphasize-lines: 4
+
+   # import json
+
+   def show_time(request):
+       values = json.loads(request.data)
+
+       dt = values.get("dt")
+       dt_naive = parser.parse(dt)
+
+       src = values.get("src")
+       src_zone = tz.gettz(src)
+
+       dst = values.get("dst")
+       dst_zone = tz.gettz(dst)
+
+       dst_dt = convert_tz(dt_naive, src_zone, dst_zone)
+       render_ctx = {
+           "src_dt": {
+               "text": dt_naive.ctime(),
+               "value": dt
+           },
+           "dst_dt": {
+               "text": dst_dt.ctime(),
+               "value": dst_dt.strftime('%Y-%m-%dT%H:%M')
+           },
+           "src_location": get_location(src),
+           "dst_location": get_location(dst),
+       }
+       return render_ctx
+
+
+The next thing is to set the renderer
+to :func:`render_json <clastic.render_json>`
+for this route:
+
+.. code-block:: python
+   :emphasize-lines: 7
+
+   # from clastic import render_json
```

Comment:
I had thought about these as "reminders" rather than actual code pieces at that point in the source file. But you're right, this is not a Python tutorial, the reader should be able to understand what to do.